### PR TITLE
Add form

### PR DESCRIPTION
**Pull request by Akhtar-Joomun**
Creating a contact form for my portfolio website by collaborating with Marc Raphael.

Files modified:
- index.html
- style.css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -274,6 +274,44 @@ <h2 class="list-title">Skills</h2>
...
+        <div class="form-container">
+            <p class="form-text">
+                I'm always interested in hearing about new projects, so if you'd like
+                to chat please get in touch.
+            </p>
+
+            <form class="form-container" action="https://formspree.io/f/xdoyzdjj" method="POST">
+                <input class="form-name" type="text" name="name" placeholder="Full name" maxlength="30" required />
+                <input class="form-email" type="email" name="email" placeholder="Email address" required />
+                <textarea class="form-text-area" name="message" cols="30" rows="10" maxlength="500">
```
Comment (mricanho) [inline index.html]: - [x] Add a required attribute here

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @Ak0706 @Marcraphael12, 

Good job so far! 🔥🔥 Your contact form has almost all the validations and you are following the original design.

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

almost there

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Akhtar-Joomun) [reply index.html]: Thank you for the review. The 'required' attribute has been added just now.

Comment (DeliceLydia) [reply index.html]: Well addressed this!

**Review by DeliceLydia — APPROVED** [review]
# Status: Approved 🎉✨

### Hi @Ak0706 , 

Congratulations! 🎉 , Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**